utils/ExampleRunner/build-all-examples-cli.js

```diff
@@ -123,12 +122,12 @@ if (configuration.examples) {
   // say name of running example
   const currentWD = process.cwd();
   // run the build for dicom image loader
-  shell.cd('../../dicomImageLoader');
-  shell.exec(`yarn run build:esm`);
-  shell.cd('../..');
-  shell.exec('yarn');
+  // shell.cd('../../dicomImageLoader');
```

Comment:
Can you remove these?

Reply:
I think i can, but haven't linked OHIF yet, let me see what happens after